network/config_test.go

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2017 The Alvalor Authors
+//
+// This file is part of Alvalor.
+//
+// Alvalor is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Alvalor is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with Alvalor.  If not, see <http://www.gnu.org/licenses/>.
+
+package network
+
+import (
+    "testing"
+    "time"
+    "go.uber.org/zap"
+    "io"
+    "github.com/stretchr/testify/assert"
+)
+
+func TestSetAddress(t *testing.T) {
+    config := DefaultConfig
```

Comment:
I still think it's dangerous to use the `DefaultConfig` here. By some coincidence, if the default value is the same as the one you set using the helper function, then you would end up with a defunct test case. If you simply use `config := Config{}`, then you are certain all values are default initialized and there is no such possibility of conflict.